src/css/base.scss

```diff
@@ -15,28 +15,62 @@ html {
 
 body {
 	height: 100vh;
-	// background-color: $goldenTan;
+	background-color: $backgroundColor;
 	line-height: 1.2;
 }
 
 header {
 display: flex;
 justify-content: space-between;
 padding: 20px;
-background-color: #72343480;
+background-color: $backgroundColor;
 align-items: center;
 }
 
+h1 {
+  font-family: $headerFont;
+	font-size: 2em;
+	color: $offWhite;
+}
+
+h2 {
+  font-family: $headerFont;
+  color: $offWhite;
+}
+
 
 .navBtn,
 .bookingBtn {
-  background-color: #364145;
-  color: #ffffff;
+  background-color: $backgroundColor;
+  color: $offWhite;
   border: none;
   padding: 10px 20px;
   border-radius: 5px;
 }
 
+.loginHere {
+  display: flex;
```

Comment:
If you're using a lot of flex, you might consider doing a mix in with display flex, the flex direction and the flex alignment.